ui/src/app/utils/auth.tsx

```diff
@@ -0,0 +1,52 @@
+// utilities around Authority and page-based authorization
```

Comment:
nit: auth usually means authentication, authorization, or both. Here I think it means authority instead, I would probably rename the file to authorities.tsx to be more explicit and avoid a third disambiguation

Reply:
good call